packages/snowpack/src/util.ts

```diff
@@ -30,6 +30,8 @@ const LOCKFILE_HASH_FILE = '.hash';
 export const HAS_CDN_HASH_REGEX = /\-[a-zA-Z0-9]{16,}/;
 // NOTE(fks): Must match empty script elements to work properly.
 export const HTML_JS_REGEX = /(<script.*?type="?module"?.*?>)(.*?)<\/script>/gms;
+export const SVELTE_VUE_REGEX = /(<script[^>]+)(.*?)<\/script>/gms;
```

Comment:
I’m thinking it’s safer to keep existing behavior for HTML—only allow `<script type="module">`, but make an exception for Vue & Svelte where all `<script>` tags are parsed as JS. Is this correct?